app/src/main/java/com/cortado/mafkir/ui/AddFragment.kt

```diff
@@ -0,0 +1,64 @@
+package com.cortado.mafkir.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.Navigation
+import com.cortado.mafkir.R
+import com.cortado.mafkir.model.MafkirContactViewModel
+import com.cortado.mafkir.model.ViewModelProviderFactory
+import dagger.android.support.DaggerFragment
+import kotlinx.android.synthetic.main.fragment_add.*
+import javax.inject.Inject
+
+class AddFragment : DaggerFragment() {
```

Comment:
specify that it adds contact or something like that, kinda confusing everywhere in the code especially since add is a verb

Reply:
btw this and EditFragment have lotsa code dup, im pretty sure they can be combined with the use of parameters

Reply:
i know they are very similair but i like the seperation